starknet_py/net/full_node_client.py

```diff
@@ -280,10 +280,11 @@ async def get_transaction_by_block_id(
         block_number: Optional[Union[int, Tag]] = None,
     ) -> Transaction:
         """
-        Get the details of transaction in block indentified block_hash and transaction index
+        Get the details of transaction in block identified block_hash and transaction index
```

Comment:
```suggestion
        Get the details of transaction in block identified by block_hash and transaction index.
```
Should there be `by` or?